### DISCUSSION TITLE
Welcome to `remix-seo` Discussions!

### DISCUSSION THREAD

**Discussion by chaance** [chaance/remix-seo#1] [Announcements]
Hello!

This is a new and experimental package. The existing API is quite simple and doesn't cover every case I'd like it to. The scope will be ironed out over time, and I'd love for you to be a part of that process. Feel free to kick the tires a bit and offer any feedback, ask questions, or whatever you need to do here in **Discussions**. 🥳

**Comment by chasoft** [comment]
Hi chaance,

I'm using `remix-seo` for all my websites. Currently, it is great to add basic meta tags, og, twitter.. for each page. But I would recommend adding more features. That is adding `<script type="application/ld+json">` inside `<head>`

It is `structured data`

> Structured data is a standardized format for providing information about a page and classifying the page content. If you're new to structured data, you can learn more about [how structured data works](https://developers.google.com/search/docs/advanced/structured-data/intro-structured-data).

Thank you!

**Reply by chasoft** [reply]
Currently, I'm using `<StructuredData />` from `remix-utils`
